### DISCUSSION TITLE
The tilte bar on the top

### DISCUSSION THREAD

**Discussion by blackcoder234** [ReVanced/revanced-discussions#1122] [Question]
In my YouTube music there is bar that show that please update the version, and the songs are not frequently being played. So please can you help what to do?

**Comment by oSumAtrIX** [comment]
Update the app

**Reply by blackcoder234** [reply]
Yes, after the update this remains still  same as before so I thought talk to the community. ![Screenshot_2024-12-16-06-37-47-410_app.revanced.android.apps.youtube.music.jpg](https://github.com/user-attachments/assets/1f20fd02-cf8f-4cf8-aa77-5e65fac30337)

**Reply by LisoUseInAIKyrios** [reply]
You need to patch version `7.29.52`

**Reply by blackcoder234** [reply]
Ok I will do,
Thanks for helping
